edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Diagnostics/publisher/EdgeRuntimeDiagnosticsUpload.cs

```diff
@@ -9,11 +9,11 @@ namespace Microsoft.Azure.Devices.Edge.Agent.Diagnostics.Publisher
     using Microsoft.Azure.Devices.Client;
     using Microsoft.Azure.Devices.Edge.Util;
 
-    public sealed class IoTHubMetricsUpload : IMetricsPublisher
+    public sealed class EventHubMetricUpload : IMetricsPublisher
```

Comment:
why filename and class out of sync?  And there is another file with same namespace and class name.

Reply:
I just changed this. Forgot to change the name. Will fix now